.github/ISSUE_TEMPLATE/release.md

```diff
@@ -59,7 +67,8 @@ Add any necessary assets to the release. They should include:
 
 The release notes should list:
 
-- The priority of the release (i.e., how quickly users should upgrade)
+- The priority of the release (i.e., how quickly users should upgrade) - this is
+    based on the max priority of any *client* changes.
```

Comment:
Indentation looks wrong here?

Reply:
Nope :) I indent on newlines that are part of the same element (i.e., when this is rendered to HTML on creation of the Issue, this will all be the same line), in order to differentiate it visually from the next list item.